src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.